### ISSUE TITLE
tax rules template understanding

### ISSUE THREAD

**Issue by lunika** [thelia/thelia#1154]
All the UI about tax and tax rule are realy hard to understand.

It would be great to refactor all this part and display more help message.

**Comment by roadster31** [comment]
:+1:

**Comment by zzuutt** [comment]
:+1:
je pense aussi qu'il faudrait modifier le fonctionnement.
Le fonctionnement actuel ne permet pas de gerer l'ecotaxe correctement.
je m'explique:
l'ecotaxe est un montant **HT** declaré grace a une caracteristique 
on applique la TVA 20% sur la somme du prix HT + ecotaxe HT

Pour un client non pro on doit lui indiquer le prix TTC **et** l'eco-participation TTC
actuellement on ne peut que l'afficher en **HT** (l'ecotaxe)

Sur la facture finale la repartition des taxes est fausse
exemple:
un produit X a **45.83 Euros HT**   + eco ** 0.17 Euros HT**  -> total taxes **9.37 Euros**   > prix TTC **55.20Euros**
si j'affiche le detail des taxes cela me donne
Ecotaxe : **0.17 Euros**
TVA francaise a 20 % : **9.20Euros**

Ce qui est faux dans la realite cela devrait etre:
Ecotaxe :  **0.204 Euros TTC**
TVA francaise a 20%: **9.17 Euros**

l'ideal aurait ete de pouvoir creer 2 regles
 - regle 1 taxe 20% : Prix du produit HT + 20%
 - regle 2 ecotaxe : motant eco + 20%
prix du produit : regle 1 + regle 2
ainsi l'affichage du detail des taxes est correct
et si l'ecotaxe un jour est soumis a un autre taux que 20% c'est possible de le modifier

j'espere avoir ete clair ;) (desole pour le francais)

**Comment by pierr0t** [comment]
+1
Et j'ajouterai: https://github.com/thelia/thelia/issues/1092

**Comment by bibich** [comment]
I close this issue because the page has been revamped in version 2.3.0-alpha1 and documentations added (maybe not enough). Concerning the tax rules I keep the issue #1092 opened because it contains a lot of interesting information.

**Comment by roadster31** [comment]
I do not agree, the Tax UI is now worse than before, barely unusable. I did not manage to create a new tax...

**Comment by bibich** [comment]
Ok, this page is still too unintelligible to ordinary people.
We have to improve UI/UX.

**Comment by anoziere** [comment]
Revisiting this 10-year-old thread: the original ask ("display more help message") has been addressed since the 2.3.0-alpha1 revamp and is still present today. The taxes/tax-rules list page explains the tax vs. tax-rule distinction with a concrete VAT/ecotax example, and the tax-rule editor's country/configuration matrix has explanatory footers describing how tax groups are applied. This content was carried over as-is into the new Twig back office as well.

The follow-up concern that the drag-and-drop tax-group editor itself is hard to use is a fair point, but it has stayed an open-ended redesign request with no concrete proposal attached since 2016. Closing; if the UX pain is still felt today, please open a fresh issue with a concrete mockup or interaction proposal for the tax-group editor so it can be scoped and estimated properly.
